src/http/client.rs

```diff
@@ -1430,6 +1431,14 @@ impl Http {
                     multipart = multipart
                         .file(file_num.to_string(), path)?;
                 },
+                AttachmentType::Image(url) => {
+                    let mut picture: Vec<u8> = vec![];
+                    let req = &mut self.client.get(url).send()?;
+                    let _ = std::io::copy(&mut *req, &mut picture);
```

Comment:
You should handle the result of `std::io::copy` here instead of ignoring it with a `let _` binding